compose/.env.es7

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=7.5.0
+ES_VERSION=7.10.0
```

Comment:
https://github.com/elastic/elasticsearch/blob/v7.10.2/LICENSE.txt 7.10.2 is the last 7.x Apache 2.0 release.